internal/sync/sync.go

```diff
@@ -129,3 +127,46 @@ func NewCommand(finders ...depbot.FinderFn) *Command {
 		finders: finders,
 	}
 }
+
+func urlOrDefaultFromArgs(args []string) string {
+	url := valueFromArgs(args, "--server-address")
+
+	if url == "" {
+		url = os.Getenv(DepbotServerAddr)
+	}
+	if url == "" {
+		url = "http://app.depbot.com/api/sync"
+	}
+
+	if !strings.HasPrefix(url, "http") {
```

Comment:
@bcuadrad1 this prefix will need to be `http://` or `https://` otherwise things like: httptie.com would fail silently.